spec/index.html

```diff
@@ -2847,7 +2837,7 @@ <h2>Privacy Considerations</h2>
       This specification has been designed to help make additional processing easier, but
       other specifications that build on top of this one are responsible for providing any
       specific details.
-      See the Data Integrity specification for more details about such post-processing methods.
+      See the [[[VC-DATA-INTEGRITY]]] [[VC-DATA-INTEGRITY]] for more details about such post-processing methods.
```

Comment:
Reference this section specifically: https://www.w3.org/TR/vc-data-integrity/#selective-disclosure